README.md

```diff
@@ -20,11 +22,13 @@ The entries should be in the following format:
 
 ## HTTP return codes
 
```

Comment:
Originally we had only one endpoint so 2 listings of request/response specs was OK. We now have 3 endpoints so I think its time we split this information on an endpoint basis - otherwise it gets too confusing.

Reply:
split into two tables